api/languages/vi.php

```diff
@@ -0,0 +1,57 @@
+<?php
+$lang = [
+  'changeEmail.footer' => 'Bạn nhận được email này vì ai đó đã cố gắng kết nối tài khoản với địa chỉ email của bạn. Trong trường hợp bạn không yêu cầu, vui lòng bỏ qua và xóa email này.',
+  'changeEmail.subject' => '$projectName: Xác nhận thay đổi địa chỉ email  ',
+  'changeEmail.body' =>
+      'Xin chào,' . "\n\n"
+    . 'bạn đã yêu cầu thay đổi địa chỉ email của $projectName account to $newEmail.' . "\n\n"
```

Comment:
It seems some lines here are not translated. Can you check?

Reply:
@pschlan thank you, i will update